zookeeper/content.md

```diff
@@ -0,0 +1,55 @@
+# What is Apache Zookeeper?
+
+Apache ZooKeeper is a software project of the Apache Software Foundation, providing an open source distributed configuration service, synchronization service, and naming registry for large distributed systems. ZooKeeper was a sub-project of Hadoop but is now a top-level project in its own right.
+
+> [wikipedia.org/wiki/Apache_ZooKeeper](https://en.wikipedia.org/wiki/Apache_ZooKeeper)
+
+%%LOGO%%
+
+# How to use this image
+
+## Start a Zookeeper server instance
+
+	$ docker run --name some-zookeeper --restart always -d 31z4/zookeeper
```

Comment:
Instances of `31z4/zookeeper` throughout should be replaced by simply `zookeeper` :+1: